.github/workflows/testapp-docker.yml

```diff
@@ -2,6 +2,8 @@ name: Docker E2E Node
 # Build & Push rebuilds the e2e Testapp docker image on every push to main and creation of tags
 # and pushes the image to https://hub.docker.com/r/tendermint/e2e-node
 on:
+  # Allow manual image builds
+  workflow_dispatch:
   push:
     branches:
       - main
```

Comment:
This actually needs to be:

```yaml
      - main
      - v0.34.x
```

on the v0.34.x branch, similar to https://github.com/informalsystems/tendermint/blob/v0.34.x/.github/workflows/tendermint-docker.yml#L10-L11

Reply:
Thanks for the catch. Proposed a fix in #17